connect/src/com/telenor/mobileconnect/MobileConnectSdkProfile.java

```diff
@@ -247,4 +268,29 @@ public void getUserInfo(
                     userInfoCallback);
         }
     }
+
+    private class OperatorDiscoveryConfigStore {
```

Comment:
Same as above, I think extracting this class into it's own file would make for better readability.